cotainr/util.py

```diff
@@ -160,3 +166,22 @@ def _print_and_capture_stream(*, stream_handle, print_dispatch):
         captured_stream.append(line)
 
     return captured_stream
+
+
+def answer_yes(input_text):
```

Comment:
`answer_yes` seems like a strange name for a function that prompts the user for a yes/no answer?

E.g. [`click` calls it `confirm()`](https://click.palletsprojects.com/en/8.1.x/prompts/#confirmation-prompts).

Reply:
Name has been changed. I don't think that "confirm" is a very good name